judge/comments.py

```diff
@@ -72,6 +72,17 @@ def post(self, request, *args, **kwargs):
         if self.is_comment_locked():
             return HttpResponseForbidden()
 
+        parent = request.POST.get('parent')
+        if parent:
+            try:
+                parent = Comment.objects.get(id=int(parent))
+            except ValueError:
+                return HttpResponseForbidden()
+            if not parent:
```

Comment:
This will actually never be executed; if there is nothing in the set, get will raise `Comment.DoesNotExist`.